config/locales/en/file_attachments/edit.yml

```diff
@@ -5,3 +5,16 @@ en:
       attachment_title:
         heading: Edit title
         hint_text: Use the official title of the document
+      attachment_file:
+        heading: Replace file
+        description_govspeak: |
+          All files must be [accessible](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/accessible-pdfs){:target="_blank"}.
+
+          Files must be uploaded in an [open standards file format](https://www.gov.uk/guidance/content-design/planning-content#open-formats){:target="_blank"}.
+          For example, .odt not .docx.
+        filename_change_callout_html: "If your new file has a different file name or file type to the original you will need to update the attachment markdown in your content.</br><p>Existing file: %{filename}</p>"
```

Comment:
It's a bit strange that the latter part of this is in a p tag but the former isn't.

Should it maybe be two paragraphs each with their own translation phrase (that'd allow them not being html)

I think the p tags should always have a class too, perhaps govuk-body but Alex can guide you better.

Reply:
It doesn't seem to be possible to pass two separate translations into the inset text component, so I'm keeping both parts in the same translation, but each wrapped in their own p tag.